lib/core/Axios.js

```diff
@@ -51,7 +51,11 @@ class Axios {
           err.stack = stack;
           // match without the 2 top stack lines
         } else if (stack && !String(err.stack).endsWith(stack.replace(/^.+\n.+\n/, ''))) {
-          err.stack += '\n' + stack
+          try {
+            err.stack += '\n' + stack
+          } catch (e) {
+            // ignore the case where "stack" is an un-writable property
+          }
         }
       }
```

Comment:
Just in case, it would be better to wrap the entire condition, not just inside the else-if branch. It will fail if someone defines this prop as 
```
{
  get() {
    return null; // undefined // etc
  },
  set() {
    throw Error('read-only');
  }
}
```
This is just a potentially improbable case, but still.